remoteworkitem/trackeritem_test.go

```diff
@@ -0,0 +1,30 @@
+package remoteworkitem
+
+import (
+	"testing"
+
+	"github.com/almighty/almighty-core/resource"
+)
+
+func TestUpload(t *testing.T) {
+	resource.Require(t, resource.Database)
+	db.Exec(`DELETE FROM "tracker_items"`)
+	tr := Tracker{URL: "https://api.github.com/", Type: "github"}
+	db.Create(&tr)
+	tq := TrackerQuery{Query: "some random query", Schedule: "0 0 0 * * *", TrackerID: tr.ID}
+	db.Create(&tq)
+	db.Delete(&tq)
+	db.Delete(&tr)
+	i := map[string]string{"content": "some text", "id": "https://github.com/golang/go/issues/124", "batch_id": "10"}
+
+	// create
+	err := upload(db, int(tq.ID), i)
+	if err != nil {
+		t.Error("Create error:", err)
+	}
+	// update
+	err = upload(db, int(tq.ID), i)
```

Comment:
@aslakknutsen there is a unique key constraint for the remote item ID, is that ok? Though I can verify the data is other fields are really saved/updated.